test/e2e/monitor_resources.go

```diff
@@ -119,7 +119,16 @@ var _ = Describe("Resource usage of system containers", func() {
 			for container, cUsage := range usage {
 				Logf("%v on %v usage: %#v", container, node, cUsage)
 				if !allowedUsage[container].isStrictlyGreaterThan(cUsage) {
-					violating[node] = usage
+					if allowedUsage[container].CPUUsageInCores < cUsage.CPUUsageInCores {
+						Logf("CPU is too high for %s (%v)", container, cUsage.CPUUsageInCores)
+					}
+					if allowedUsage[container].MemoryUsageInBytes < cUsage.MemoryUsageInBytes {
+						Logf("Memory use is too high for %s (%v)", container, cUsage.MemoryUsageInBytes)
+					}
+					if allowedUsage[container].MemoryWorkingSetInBytes < cUsage.MemoryWorkingSetInBytes {
+						Logf("Working set is too high for %s (%v)", container, cUsage.MemoryWorkingSetInBytes)
+					}
+					violating[node][container] = usage[container]
```

Comment:
this will cause panics, since `violating` is a `map[string]resourceUsagePerContainer`, and `violating[node]` resolves to nil:

```
Oct  9 00:32:33.902: INFO: CPU is too high for /docker-daemon (0.9293734804884075)

•! Panic [15.601 seconds]
Resource usage of system containers
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/monitor_resources.go:138
  should not exceed expected amount. [It]
  /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/monitor_resources.go:137

  Test Panicked
  assignment to entry in nil map
  /usr/src/go/src/runtime/panic.go:387

  Full Stack Trace
  /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/e2e_test.go:140 (0x4f0f6e)
  /usr/src/go/src/testing/testing.go:447 (0x460b0f)
  /usr/src/go/src/runtime/asm_amd64.s:2232 (0x440671)
```